crates/query-engine/translation/src/translation/query/filtering.rs

```diff
@@ -368,27 +410,18 @@ fn translate_comparison_value(
     state: &mut State,
     root_and_current_tables: &RootAndCurrentTables,
     value: models::ComparisonValue,
-    typ: &database::ScalarType,
+    typ: database::Type,
```

Comment:
These could probably both be some references to avoid clones at the call sites.

Reply:
Done. Thanks